### PR TITLE
[zh-cn] Some cleanup in tutorials to sync with en pages

### PR DESCRIPTION
**Pull request by fenggw-fnst**

### CODE REVIEW

File: content/zh-cn/docs/tutorials/stateful-application/cassandra.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ kubectl apply -f cassandra-statefulset.yaml
    -->
    响应应该与此类似：
 
-   ```
+   ```shell
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh-cn/docs/tutorials/stateful-application/cassandra.md:252]: Thanks for the review, but your "Suggested change" above is `shell`?
Should I change to `none` or `shell`?

[the remark this answers is not in the document]
Comment (windsonsea) [reply content/zh-cn/docs/tutorials/stateful-application/cassandra.md:252]: `none`

[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh-cn/docs/tutorials/stateful-application/cassandra.md]: Thanks, done.

**Comment by windsonsea** [comment]
You may need to confirm these 3 files are completely synced and consistent with en version rather than only picking a trival change. A suggesion is to pick a page and read it through, which can inspire you much where can be improved including:
- text
- format
- link
- more fluent and natural

image

**Comment by fenggw-fnst** [comment]
Thanks for the review and sorry for my late response.

One of my change has been updated by https://github.com/kubernetes/website/pull/40526, and the change in `cassandra.md` is actually an en page mistake, so this PR doesn't fix much and I want to close it.

My original intention was to fix/sync zh-cn pages from the command interpreter mark aspect, because I've found a lot of pages ignoring this when updating the page...

**Pull request closed without merging**